card/lib/card/format.rb

```diff
@@ -392,9 +392,12 @@ def prepare_nest opts
 
       opts ||= {}
       case
-      when opts.has_key?( :comment )                            ; opts[:comment]   # as in commented code
-      when @mode == :closed && @char_count > Card.config.max_char_count   ; ''     # already out of view
-      when opts[:inc_name]=='_main' && !Env.ajax? && @depth==0  ; expand_main opts
+      when opts.has_key?( :comment )                                      ; opts[:comment]
+        # commented nest
+      when @mode == :closed && @char_count > Card.config.max_char_count   ; ''
+        # already out of view
+      when opts[:inc_name]=='_main' && show_layout? && @depth==0          ; expand_main opts
+        # why do we have these restrictions on main?
```

Comment:
Yes, but the comment asks why we need to test more than just _main, and I think it is because of what you write.  We only want this particular processing when it is the main card, always depth 0 from a layout card.
And I guess you are saying there is another place where inclusion of _main is suppressed (when the layout is the main card, for example).  Subbing in the _main content is a different step when processing the nest at the next level.